server/src/test/java/org/elasticsearch/search/aggregations/bucket/composite/CompositeValuesCollectorQueueTests.java

```diff
@@ -133,9 +133,9 @@ public void testRandom() throws IOException {
     }
 
     private void testRandomCase(ClassAndName... types) throws IOException {
-        testRandomCase(true, true, types);
+        //testRandomCase(true, true, types);
         testRandomCase(true, false, types);
-        testRandomCase(false, true, types);
+        //testRandomCase(false, true, types);
```

Comment:
why is it disabled?

Reply:
it's a leftover, I  enabled these tests in https://github.com/elastic/elasticsearch/pull/36675/commits/b80398874b10bfce242373efe8b7f997ff610694